packages/bot-web-ui/src/pages/tutorials/quick-strategy-content/index.scss

```diff
@@ -21,7 +21,7 @@
         grid-template-columns: 10fr 1fr;
         align-items: center;
         padding: 1.6rem;
-        border: 2px solid var(--system-light-5-active-background, #d6dadb);
+        border: 0.2rem solid var(--border-normal);
```

Comment:
@farabi-deriv Using **px** as the **unit** for defining **border** thickness in CSS is a best practice for ensuring predictability, control, and performance in your design. It provides precise and consistent border display across various devices and browsers, gives full control over border thickness without unwanted scaling issues, and can improve rendering performance.

Reply:
Sure @maryia-matskevich-deriv